apps/crn-server/src/utils/arrays.ts

```diff
@@ -0,0 +1,8 @@
+export const getFirstOrAll = <T>(
```

Comment:
this util is not tested, nor does it seem particularly helpful, the output type seems very much connected to the filter tool itself. I'm not sure what the purpose of it is but i would at least move it to the odata filter util